fixtures/large/rspec_mocks_proxy_expected.rb

```diff
@@ -390,7 +392,7 @@ def initialize(source_space, *args)
       # That's what this method (together with `original_unbound_method_handle_from_ancestor_for`)
       # does.
       def original_method_handle_for(message)
-        unbound_method = superclass_proxy && superclass_proxy.original_unbound_method_handle_from_ancestor_for(message.to_sym)
```

Comment:
it is correct for standard to make this change, it's not a change rubyfmt can make